website/docs/r/integration_aws_gov_cloud_cfg.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Cloud Account Integrations"
+layout: "lacework"
+page_title: "Lacework: lacework_integration_aws_gov_cloud_cfg"
+description: |-
+  Create and manage AWS GovCloud Config integrations
+---
+
+# lacework\_integration\_aws\_gov\_cloud\_cfg
+
+Use this resource to configure an AWS GovCloud Config integration to analyze AWS configuration compliance.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360021140214-Initial-Setup-of-AWS-GovCloud-Integration).
+
+## Example Usage
+
+```hcl
+resource "lacework_integration_aws_gov_cloud_cfg" "example" {
```

Comment:
Rename to lacework_integration_aws_govcloud_cfg ?

Reply:
Definitely! 💯